lib/highline.rb

```diff
@@ -677,20 +449,17 @@ def indentation
   def indent(increase=1, statement=nil, multiline=nil)
     @indent_level += increase
     multi = @multi_indent
-    @multi_indent = multiline unless multiline.nil?
+    @multi_indent = multiline || @multi_indent
```

Comment:
I couldn't agree more. Thank you @plribeiro3000 

Could you do a commit so I could cherry-pick and you could keep the "credits" for the change?

But if you don't want to, just let me know that I can make the commit with the change.

Thank you again.